website/docs/docs/collaborate/explore-projects.md

```diff
@@ -53,17 +53,23 @@ To interact with the full lineage graph, you can:
 - Hover over any item in the graph to display the resource’s name and type.
 - Zoom in and out on the graph by mouse-scrolling.
 - Grab and move the graph and the nodes.
+- Right click on a node (context menu) to:
+    - Refocus on the node, including its parent and child nodes
+    - Refocus on the node and its children only
+    - Refocus on the node and it parents only
+    - View the node's [resource details](#view-resource-details) page
+
 - Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details.
     - Click the Share icon in the side panel to copy the graph’s link to your clipboard.
     - Click the View Resource icon in the side panel to [view the resource details](#view-resource-details). 
-- [Search and select specific resources](#search-resources) or a subset of the DAG using selectors and graph operators. For example:
+- [Search and select specific resources](#search-resources) or a subset of the DAG using [selectors](/reference/node-selection/methods) and [graph operators](/reference/node-selection/graph-operators). This can help you narrow the focus on the resources that interest you. For example:
     - `+[RESOURCE_NAME]` &mdash; Displays all parent nodes of the resource
```

Comment:
non blocking sugg - i like the example and i would suggest sharing a real-life example that users new to explorer will immediately understand, as opposed to only displaying a generic 'resource_type:model [resource_name]` example.

Reply:
similar to how you display an example in 'Search with selectors'